lib/src/core/better_player_controller.dart

```diff
@@ -1181,14 +1187,17 @@ class BetterPlayerController {
         !(videoPlayerController?.value.hasError ?? true);
 
     if (isPipSupported && canEnablePictureInPicture) {
-      if (isFullScreen) exitFullScreen();
+      _isPip = true;
+      if (Platform.isIOS) exitFullScreen();
       _wasInFullScreenBeforePiP = _isFullScreen;
       _wasControlsEnabledBeforePiP = _controlsEnabled;
       setControlsEnabled(false);
       if (Platform.isAndroid) {
+        if (isFullScreen) exitFullScreen();
```

Comment:
nie do końca rozumiem jaka jest tutaj różnica, gdy `if (isFullScreen) exitFullScreen();` daje nam to samo co: `if (Platform.isIOS) exitFullScreen();` i. `if (isFullScreen) exitFullScreen();` tylko rozbite jest na dwa warunki. Chodzi o kolejność ustawiania tych flag, czy te setControlsEnabled ?